test/onigumo_test.exs

```diff
@@ -24,12 +24,14 @@ defmodule OnigumoTest do
 
   @tag :tmp_dir
   test("load URL from file", %{tmp_dir: tmp_dir}) do
+    input_urls = [@url]
```

Comment:
```suggestion
```

Reply:
This is no longer valid, because `@url` ceased to exist and `@urls` contain more than one entry. I went for `Enum.slice(@urls, 0, 1)` to pick the first one.